mongosql/src/mir/optimizer/stage_movement/test.rs

```diff
@@ -140,6 +140,119 @@ macro_rules! test_move_stage_no_op {
     };
 }
 
+test_move_stage!(
```

Comment:
Let's use the `test_move_stage_no_op!` macro for any tests that expect the `expected` to be the same as the `input`. This not only reduces the size of the test file but also -- in addition to the test name -- helps make it clear that the test is asserting movement _does not_ happen.

Reply:
Ok I see this is the only test in this PR that would require this change! Feel free to just change this one -- no need to audit the rest of the test file for `expected_changed = false`. We can do that another time.